tower-balance/Cargo.toml

```diff
@@ -41,7 +42,7 @@ hdrsample = "6.0"
 quickcheck = { version = "0.6", default-features = false }
 tokio = "0.1.7"
 tokio-executor = "0.1.2"
-tower = { version = "0.1", path = "../tower" }
-tower-buffer = { version = "0.1", path = "../tower-buffer" }
-tower-limit = { version = "0.1", path = "../tower-limit" }
-tower-test = { version = "0.1.0", path = "../tower-test" }
+tower = { version = "*", path = "../tower" }
+tower-buffer = { version = "*", path = "../tower-buffer" }
+tower-limit = { version = "*", path = "../tower-limit" }
+tower-test = { version = "*", path = "../tower-test" }
```

Comment:
@carllerche like this?

Reply:
Yes, that looks like what I had in mind.

Full disclosure: I haven't actually updated any other project ot do this... so this will be the experiment!